src/grpc/infer_handler.h

```diff
@@ -299,7 +299,9 @@ InferAllocatorPayload(
     const inference::ModelInferRequest& request,
     std::list<std::string>&& serialized_data,
     std::shared_ptr<ResponseQueue<ResponseType>> response_queue,
-    AllocPayload<ResponseType>* alloc_payload)
+    AllocPayload<ResponseType>* alloc_payload,
+    std::vector<std::shared_ptr<const SharedMemoryManager::SharedMemoryInfo>>*
+        ref_shm_regions)
```

Comment:
❓ From a naming point of view we do not typically use the prefix "ref" - do we need it here? Or could these be ```shm_regions``` and elsewhere ```shm_info```

Reply:
Thanks @nnshah1 for the note. I updated the names accordingly to `shm_regions_info`, and `shm_info`.